_episodes/03-what-is-open-source.md

```diff
@@ -167,6 +169,9 @@ And you can finesse license compatibility issues by letting the end user put eve
 OSI approves open-source licenses for *software.*
 But there are many other creative works related to software (or not) that we might want to treat similarly (like documentation for your software, or this lesson).
 The [Creative Commons](https://creativecommons.org/) (CC) is a family of licenses analogous to open-source but for things other than software.
+Another class of work that can fall under Creative Commons licenses is data.
```

Comment:
Do you want to add a bit of explanation as to why?  My take would be that the SA (share-alike) clause is similar to a copyleft license in that it requires derivatives to be shared under the same license as the original.